inrupt/pod-browser
# use expiration date from access request

## Description

In this PR, we use the expiration date from the access request instead of defaulting to Forever without checking if an expiration date is available.

## Testing

- Go to [test-app-access-grants-inrupt.vercel.app](https://test-app-access-grants-inrupt.vercel.app/) and generate a request (it will have an expiration date)
- Grab the URL for the request and append to the path for the URL of the Vercel deployment for this PR
- Verify a date is displayed (today's - the access request is for a few minutes)

## Commit checklist
